mysql-test/suite/tianmu/t/issue1187.test

```diff
@@ -0,0 +1,40 @@
+--source include/have_tianmu.inc
+
+--disable_warnings
+DROP DATABASE IF EXISTS issuse1187_test;
+--enable_warnings
+
+CREATE DATABASE issuse1187_test;
+USE issuse1187_test;
+
+create table t1(id int,name varchar(5)) ;
+insert into t1 values(1,'AAA'),(2,'BBB');
+alter table t1 add column age int null;
+select * from t1;
+
+create table t2(id int,name varchar(5)) ;
+insert into t2 values(1,'AAA'),(2,'BBB');
+alter table t2 add column age int not null;
+select * from t2;
+
+create table t3(id int,name varchar(5)) ;
+insert into t3 values(1,'AAA'),(2,'BBB');
+alter table t3 add column age int not null default 66;
```

Comment:
F.I.Y: Just only change the default value.
``ALTER TABLE table_name ALTER COLUMN column_name SET DEFAULT 'literal';``

Reply:
1. ok, i will try to add some wrong default value; maybe it will be intercepted by sql layer, i will test it.
2. "Just only change the default value" is a related sceanrio, i will add this MTR.